static/app/views/alerts/rules/metric/triggers/actionsPanel/actionTargetSelector.tsx

```diff
@@ -87,7 +87,6 @@ export default function ActionTargetSelector(props: Props) {
           type="text"
           autoComplete="off"
           disabled={disabled}
-          required={action.type === 'discord'} // Only required for discord channel ID
```

Comment:
Not sure who to tag here, @evanpurkhiser or @mifu67 since you seem to have made some changes here